README.md

```diff
@@ -21,7 +21,82 @@ pip install -e .
 
 ## Usage
 
-See the [tutorial notebook](https://github.com/rly/ndx-beadl/blob/main/docs/tutorial_nwb_userdays_2022/beadl_example_nwb_userdays_2022.ipynb) for examples. 
+```python 
+from ndx_beadl import (Task, BEADLTaskProgram, BEADLTaskSchema, EventTypesTable, EventsTable,
+                       StateTypesTable, StatesTable, TrialsTable, ActionTypesTable, ActionsTable, TaskArgumentTable)
+from ndx_beadl.beadl_xml_parser import BeadlXMLParser
+from pynwb import NWBFile, NWBHDF5IO
 
+# Import the BEADL task schema and task program
+with open(src/pynwb/tests/beadl_schema_file, "r") as test_xsd_file:
+    test_xsd = test_xsd_file.read()
+
+with open(src/pynwb/tests/beadl_task_file, "r") as test_xml_file:
+    test_xml = test_xml_file.read()
```

Comment:
```suggestion
with open("src/pynwb/tests/beadl_schema_file", "r") as test_xsd_file:
    test_xsd = test_xsd_file.read()

with open("src/pynwb/tests/beadl_task_file", "r") as test_xml_file:
    test_xml = test_xml_file.read()
```

Reply:
Please try and copy the script to a .py file to make sure it runs. Also please add a unit test for the script to make sure the example from the README is running, since this is what folks will likely copy as a starting point.